core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -73,10 +93,29 @@ public void initializeLogging(LogConfig config) {
         InitialConfigurator.DELAYED_HANDLER.setHandlers(handlers.toArray(EmbeddedConfigurator.NO_HANDLERS));
     }
 
+    private boolean workingConsole() {
```

Comment:
Maybe it's not the right name? It's just about color support, not if the console is working or not.

Reply:
I suggest: `hasColorSupport` or `supportsColors`

Reply:
fixed.